reconcile/aus/metrics.py

```diff
@@ -0,0 +1,72 @@
+from pydantic import BaseModel
+
+from reconcile.utils.metrics import (
+    CounterMetric,
+    GaugeMetric,
+    InfoMetric,
+)
+
+
+class AUSBaseMetric(BaseModel):
+    "Base class for AUS metrics"
+
+    integration: str
+    ocm_env: str
+
+
+class AUSClusterVersionRemainingSoakDaysMetric(AUSBaseMetric, GaugeMetric):
+    "Remaining days a version needs to soak for a cluster"
+
+    cluster_uuid: str
+    soaking_version: str
+
+    @classmethod
+    def name(cls) -> str:
+        return "aus_cluster_version_remaining_soak_days"
+
+
+class AUSClusterUpgradePolicyInfoMetric(AUSBaseMetric, InfoMetric):
+    "Info metric for clusters under AUS upgrade control"
+
+    cluster_uuid: str
+    org_id: str
+    cluster_name: str
+    schedule: str
+    sector: str
+    mutexes: str
+    soak_days: str
+    workloads: str
+
+    @classmethod
+    def name(cls) -> str:
+        return "aus_cluster_upgrade_policy_info"
+
+
+class AUSOrganizationValidationErrorMetric(AUSBaseMetric, GaugeMetric):
```

Comment:
don't you want to name that according to the metric type?  `AUSOrganizationValidationErrorGauge` ? Maybe also add the `s` at `Error` ?

The same applies to `AUSClusterVersionRemainingSoakDaysMetric` -> `AUSClusterVersionRemainingSoakDaysGauge` above

Reply:
good point. fixed